tests/templates/_nav.html

```diff
@@ -1,6 +1,8 @@
-{%- if current_user.is_authenticated -%}
-  <p>Hello {{ current_user.email }}</p>
```

Comment:
Maybe we can register new filter: `security_user_attribute` so we can do something like: `{{ current_user|security_user_attribute }}`.